exs/mundo_1/javascript/033.js

```diff
@@ -0,0 +1,19 @@
+const input = require('readline-sync').question //Chamando o modulo readline-sync para entrada de dados
+
+//Entrada de dados
+let n1 = input("Digite um primeiro número: ")
+let n2 = input("Digite um segundo número: ")
+let n3 = input("Digite um terceiro e último número: ")
+
+
+
+let max = Math.max(n1, n2, n3); //Recomendo pesquisa sobre a max e min no doc do java script.
```

Comment:
Remova o espaço do nome da lang

```suggestion
let max = Math.max(n1, n2, n3); //Recomendo pesquisa sobre a max e min no doc do javascript.
```